docs/generating_asts.rst

```diff
@@ -105,6 +114,29 @@ Parameters used by the random SED selection method
   Number of models to pick per age
 
 
+  Parameters used by the supplementing AST method
+  --------------------------------------------------
```

Comment:
This line has to be the same length as the text above.  The current line is too long and this generates an error when building the docs.

Reply:
Done. I also changed the quotation mark, which also might be causing an error.